make/compiler/Makefile.gnu

```diff
@@ -279,6 +279,13 @@ ifeq ($(shell test $(GNU_GPP_MAJOR_VERSION) -eq 8; echo "$$?"),0)
 WARN_CXXFLAGS += -Wno-class-memaccess
 endif
 
+#
+# Avoid build aborting due to this warning, it may be coming from LLVM headers
+#
+ifeq ($(shell test $(GNU_GPP_MAJOR_VERSION) -ge 8; echo "$$?"),0)
+WARN_CXXFLAGS += -Wno-error=class-memaccess
+endif
```

Comment:
This could be fine as-is but I think I have a slight preference to also use this conditional to handle GCC 8 (meaning that we could remove the conditional just before that adds `-Wno-class-memaccess`. I would think disabling the warning-as-error for this case should be sufficient. Unless, of course, `-Wno-error=class-memaccess` doesn't work for GCC 8, in  which case you'd have to adjust the `-ge` to instead be greater than.